dojson/cli/__init__.py

```diff
@@ -31,7 +35,16 @@
 .. code-block:: console
 
     $ dojson -l marcxml -d marcxml do marc21 do to_marc21 < in.xml > out.xml
```

Comment:
Sure. I used the `in.xml` as the input file doesn't really matter (on the contrary to the example with `missing` function, where it makes sense to point user to an input file with fields that are non-marc21 standard)